server/tracedb/jaegerdb.go

```diff
@@ -49,9 +49,17 @@ func (jtd *jaegerTraceDB) Connect(ctx context.Context) error {
 }
 
 func (jtd *jaegerTraceDB) TestConnection(ctx context.Context) ConnectionTestResult {
-	// TODO: when the test fails, we should still keep all messages.
-	// Current implementation makes only errors to show, so if everything worked but the last
-	// step, there's no feedback that tracetest was able to ping the service and authenticate to it.
+	connectionTestResult := ConnectionTestResult{
+		ConnectivityTestResult: ConnectionTestStepResult{
+			OperationDescription: fmt.Sprintf(`Tracetest connected to "%s"`, jtd.config.Endpoint),
+		},
+		AuthenticationTestResult: ConnectionTestStepResult{
+			OperationDescription: `Tracetest managed to authenticate with Jaeger`,
+		},
+		TraceRetrivalTestResult: ConnectionTestStepResult{
```

Comment:
Small thing to refactor in the future: this property name should be `TraceRetrievalTestResult` instead of `TraceRetrivalTestResult`. (I can do this, I'm planning to open a PR to solve these small typos that I've found)